inc/class-upgrade.php

```diff
@@ -42,6 +42,13 @@ public function __construct() {
 			$this->upgrade_22();
 		}
 
+		/**
+		 * Filter: 'wpseo_run_upgrade' - Runs the upgrade hook which are dependent on Wordpress SEO by Yoast
+		 *
+		 * @api string - The current version of Wordpress SEO by Yoast
```

Comment:
Nope, it shouldn't. The first argument will be `@api` and every extra argument will be `@param`. @omarreiss Can you confirm?